crates/pop-parachains/src/up/mod.rs

```diff
@@ -206,6 +216,18 @@ impl Zombienet {
 				continue;
 			}
 
+			// Check if command references a parachain template binary without a specified path
+			// (e.g. Polkadot SDK parachain template)
+			if ["parachain-template-node", "substrate-contracts-node"].contains(&command.as_str()) {
+				for profile in Profile::VARIANTS {
```

Comment:
Low risk, but could result in some frustration when a debug binary is being used instead of a release, as intended by the caller. Unlikely to cause unforeseen issues so expect it is fine as is.